cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -154,13 +172,21 @@
     mount options=(rw rslave) -> /tmp/snap.rootfs_*/run/,
 
     mount options=(rw rbind) {/usr,}/lib/modules/ -> /tmp/snap.rootfs_*/lib/modules/,
+    mount options=(rw rbind) {/usr,}/lib64/modules/ -> /tmp/snap.rootfs_*/lib/modules/,
     mount options=(rw rslave) -> /tmp/snap.rootfs_*/lib/modules/,
 
     mount options=(rw rbind) /var/log/ -> /tmp/snap.rootfs_*/var/log/,
     mount options=(rw rslave) -> /tmp/snap.rootfs_*/var/log/,
 
     mount options=(rw rbind) /usr/src/ -> /tmp/snap.rootfs_*/usr/src/,
     mount options=(rw rslave) -> /tmp/snap.rootfs_*/usr/src/,
+
```

Comment:
I'm curious about those two rules, they look fine but I wasn't aware we need them or that we perform those operations.